apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -424,16 +425,27 @@ function initializeBlocklyWrapper(blocklyInstance) {
         },
         null
       );
-      container.appendChild(svg);
+
+      // Core Blockly requires a container div to be LTR, regardless of page direction.
+      const div = document.createElement('div');
```

Comment:
Probably a naive idea...could we set the direction, etc on the container div that is passed in, rather than creating a new one?

Reply:
Separately, could you explain a little more (either here or in the PR description) why this solves the issue we were seeing?

Reply:
You were totally right. The extra `div` wasn't needed.